cloud/azure/app-gateway/detectors-app-gateway.tf

```diff
@@ -0,0 +1,255 @@
+resource "signalfx_detector" "heartbeat" {
```

Comment:
Use `lasting` function instead of aggregation function for Application Gateway detectors.